src/multi_vector_simulator/utils/__init__.py

```diff
@@ -158,10 +175,28 @@ def compare_input_parameters_with_reference(folder_path, ext=JSON_EXT):
 
             for sp in not_matching_params:
                 if sp in required_parameters[mp]:
-                    # the sub parameter is not provided but is required --> missing
-                    param_list = missing_parameters.get(mp, [])
-                    param_list.append(sp)
-                    missing_parameters[mp] = param_list
+
```

Comment:
I feel like this section is already in `A1.check_for_official_extra_parameters` for the CSV (we are setting default values there), and the functions are pretty much the same. We should mention that some simplifications should be possible here in #750.

Reply:
Done